app/lib/activitypub/activity/create.rb

```diff
@@ -372,7 +372,8 @@ def in_reply_to_uri
   end
 
   def converted_text
-    linkify([@status_parser.title.presence, @status_parser.spoiler_text.presence, @status_parser.url || @status_parser.uri].compact.join("\n\n"))
+    title = "<h2>#{@status_parser.title}</h2>" if @status_parser.title.present?
```

Comment:
Mastodon replaces `h1` to `h6` tags with `p` tags, so that is not especially useful.

Reply:
I thought `<h1>` to `<h6>` tags would be replaced by `<p><strong>` tags. Thats what I found out by just comparing status, e.g. [this one](https://event-federation.eu/2023/12/20/it-is-all-about-the-community/). Maybe that processing happens already before: then of course that line should be changed accordingly.

Reply:
Right, `<h2>Title</h2>` is changed to `<p><strong>Title</strong></p>`, which I guess is ok.

Reply:
Thanks again a lot for reviewing! If that is the case, I would propose to revert https://github.com/mastodon/mastodon/commit/10c768c6906de7ca70ae890f99e66669eb00202 because some forks may use the `<h>` tags. Also, if in the future there are features that are affected by these changes, I think this would be more clear.

Reply:
Done. As you mentioned no further aspects I'll close the conversation for now.